### PR TITLE
Fix texts position on wait and print backgrounds

### PR DESCRIPTION
**Pull request by anxuae**
This patch ensure that arrows are always outside of the text boxes.

@werdeil can you check please ?

### CODE REVIEW

**Comment by anxuae** [comment]
Resolution of screen on the photo box: (1440, 900)

File: pibooth/view/background.py [hunk 1]
```diff
@@ -90,7 +90,7 @@ def _write_text(self, text, rect=None, align='center'):
             rect = self._rect.inflate(-self._text_border, -self._text_border)
         if self._outlines:
             # Build rectangle around text area for debuging purpose
-            outlines = pygame.Surface(rect.size)
+            outlines = pygame.Surface(rect.size, pygame.SRCALPHA, 32)
```
Comment (werdeil) [inline pibooth/view/background.py:93]: Can you tell me what is this change? I tried with and without it and can't see the diff...

Comment (anxuae) [reply pibooth/view/background.py:93]: Use the debug mode with a background image, and admire the improvement ;-)

**Comment by anxuae** [comment]
To see the changes, put your config in debug mode.
Don't use the full screen and resize your screen. The arrow will never go over the text.

**Comment by werdeil** [comment]
Still can't see the impact as I have the same resizing behavior on my machine with or without the pygame.SRCALPHA but anyway I merge it

**Pull request merged**